{
  try {
    this.fault.setFaultCode(code);
  }
 catch (  SOAPException e) {
    throw new RuntimeException("Error creating fault message.",e);
  }
  return this;
}
